scripts/common_windows.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# 	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+set -uo pipefail
+
+#############################################################################################################
+# Helper methods to set the required variables.
+#############################################################################################################
+# Constant
+# Number of presently supported OS versions.
+SUPPORTED_WINDOWS_OS_RELEASES_NUMBER=2
```

Comment:
What does this mean?

Reply:
So this is essentially the count of number of supported Windows releases. Right now it is 2 i.e. Windows Server 2019 and Windows Server 2022.

In `check_image_manifest_exists` method, we check if the image manifest for the given version does exist in the target registry and the number of digests is equal to this number i.e. 2.